docs/topics/mta-rn-known-issues-7-0-2.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * docs/release_notes/master.adoc
+
+:_content-type: REFERENCE
+[id="rn-known-issues-7-0-2_{context}"]
+= Known issues
+
+{ProductFullName} version 7.0.2 has the following known issues.
+
+.Eclipse IDE: {ProductShortName} Run_configuration not functioning as expected
```

Comment:
Please remove this as the Eclipse IDE will not be published until 7.0.3. So for known issues we can simply mention that the Eclipse IDE is still not available in the marketplace and that customers can still use the 6.2.2 version.